UIforETW/UIforETWDlg.cpp

```diff
@@ -1101,7 +1909,8 @@ void CUIforETWDlg::OnLbnDblclkTracelist()
 {
 	int selIndex = btTraces_.GetCurSel();
 	// This check shouldn't be necessary, but who knows?
-	if (selIndex < 0 || selIndex >= (int)traces_.size())
+	//That comment IS WRONG! CListBox::GetCurSel MAY return -1! (LB_ERR, when no item is selected)

```

Comment:
It is true that in general GetCurSel can return -1, but it cannot do this in OnLbnDblclkTracelist, which is called when the user double-clicks an item in the list, an it?